src/app/auth/register/register.component.ts

```diff
@@ -1,16 +1,74 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, ViewChild } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { CustomValidators } from '../services/customValidators';
+import { UserService } from '../services/user.service';
+import { AuthService } from '../services/auth.service';
+import { ModalDirective } from 'ngx-bootstrap/modal';
 
 @Component({
-  selector: 'app-register',
   templateUrl: './register.component.html',
   styleUrls: ['./register.component.scss'],
 })
-export class RegisterComponent implements OnInit {
+export class RegisterComponent {
+  public frmSignup: FormGroup;
```

Comment:
again form not frm, and name it register not signup, cause both the file name and class names are register
```suggestion
  public registerForm: FormGroup;
```